dashboard/app/dsl/script_dsl.rb

```diff
@@ -29,6 +29,7 @@ def initialize
     @project_sharing = nil
     @curriculum_umbrella = nil
     @tts = false
+    @deprecated = false
```

Comment:
I don't know enough about scripts to know if it matters, but it looks like other PLC-specific attributes (professional_learning_course, peer_reviews_to_complete) aren't in here.

Reply:
To your point: Judging by the way the spec was written, this will be a more generic attribute. I think with that in mind, I'm going to leave it in here!